### ISSUE TITLE
Add recipe for database schema documentation

### ISSUE THREAD

**Issue by daveaglick** [statiqdev/Statiq#446]
Idea is shamelessly stolen from https://github.com/schemadoc/schemadoc-theme, but it's a cool one. Use a setting to pass a connection string and then generate full database schema documentation.

**Comment by adamclifford** [comment]
Could we also have the option to target a SQL project, as we store our DB schema in source control and it would be good to have the documentation derived from that.

**Comment by daveaglick** [comment]
@adamclifford Good idea - if one recipe doesn't cover both cases, we can always make a second.
